packages/ng/user/picture/user-picture.component.html

```diff
@@ -1,3 +1,11 @@
-<div class="picture" [ngStyle]="style">
-	<span class="userPicture-initials" *ngIf="!hasPicture">{{ initials }}</span>
-</div>
+<ng-container *ngIf="!!hasPicture; then displayPicture; else displayInitials"></ng-container>
+
+<ng-template #displayPicture>
+	<img class="picture" [src]="pictureHref" [alt]="initials" (error)="pictureError()" />
```

Comment:
(Not directly related to PR, and no simple solution to provide) but I think this alt is really not interesting.

Reply:
@vvalentin-lucca, so do you recommend to remove it or to add something like : "Firstname Lastname picture" ?

Reply:
It's usually next to the person's name, so with a11y in mind, the alt is not really interesting and IMHO the img should be ignored by screen readers